packages/livecd/grub2/build.yaml

```diff
@@ -8,7 +8,7 @@ prelude:
 - zypper in -y --no-recommends syslinux grub2-i386-pc grub2-x86_64-efi
   {{end}}
   {{if eq .Values.arch "aarch64"}}
-- zypper in -y --no-recommends syslinux-x86_64 grub2-i386-pc grub2-arm64-efi
+- zypper in -y --no-recommends grub2-arm64-efi
```

Comment:
I have not tested the arm64 ISO yet, but I resist to accept we need syslinux-x86_64 and grub2-i386 on arm64.

Reply:
Huh ? Isn't the syslinux rpm just named `syslinux` ? Where's `syslinux-x86_64` coming from ?

Reply:
cant build an hybrid iso from an arm64 os then no? arent the needed files in the syslinux package and i386 packages, thus we also need them on x86_64? I mean, we are mixing x86_64 with 386 and the only reason is that we need files from those package AFAIk? Otherwise if we can drop it from here, we can drop it from the x86_64 package. 

We need to make sure that x86 and arm64 packages have exactly the same files if we support both arches